### DISCUSSION TITLE
JCB update doesn't work

### DISCUSSION THREAD

**Discussion by dormasta** [joomengine/Joomla-Component-Builder#953] [Q&A] [answered]
Hello,

I restarted an old project after a while. For no obvious reason JCB is blocked at version 3.1.2. 
Joomla is 3.10.11 and PHP is 7.4.33 and "Update Sites" is active. I went through the option I didn't found an update blocking.
Did I misconfigured something ?
Thanks for help.
Cheers
DTA

**Comment by Llewellynvdm** [comment, marked as answer]
We had a few changes and there may be an issue, best solution is to do a manual update if the auto update does not work.

JCB has moved forward and the place to watch is https://git.vdm.dev/joomla/Component-Builder

We are running tests as we speak... and plan to release JCB v3.1.24 in the next few days. JCB is now in a package to insure all plugin are also installed, and the package repository can be found here: https://git.vdm.dev/joomla/pkg-component-builder

JCB also has a beta repository where we now deploy and test all new and changing things in JCB, it can be found here: https://git.vdm.dev/joomla-beta

The beta testing has its own package release found here: https://git.vdm.dev/joomla-beta/pkg-component-builder

I am a little behind on the video tutorials but hope to start working on that again soon... JCB has grown tremendously in its transition to Joomla 4. So keep informed of all things JCB by following this channel... we are planning to build it in to JCB soon... so you will not need Telegram to read the thread. But for now: https://t.me/s/joomla_component_builder

Then to open any issue directly related to JCB core: https://git.vdm.dev/joomla/Component-Builder/issues

The fact that the auto updater is not working happened due to changes in JCB probably. The update server that is the correct update server is this one: https://git.vdm.dev/joomla/Component-Builder/raw/branch/master/componentbuilder_update_server.xml
![image](https://user-images.githubusercontent.com/5607939/235438464-80aa730e-ceaa-48ba-902b-894ecf14fc19.png)
